app/Http/Controllers/EntityCreatorController.php

```diff
@@ -76,8 +83,13 @@ public function form($type)
     public function store(Request $request, $type)
     {
         // Make sure the user is allowed to create this kind of entity
-        $class = $this->entityService->getClass($type);
-        $this->authorize('create', $class);
+        if ($type != 'posts') {
```

Comment:
Just for consistency, let's keep the same test context from in the `form()` method (checking == 'post' first)